kuma/wiki/content.py

```diff
@@ -140,7 +141,7 @@ def code_sample(self, name):
             sample = pq('<section>%s</section>' % section)
         else:
             # If no section, fall back to plain old ID lookup
-            sample = pq(src).find('[id="%s"]' % name)
+            sample = pq(src).find('[id=%s]' % quoteattr(name))
```

Comment:
Interesting use of ``xml.sax.saxutils.quoteattr``. That will probably catch many corner cases we. haven't thought of